system/index.js

```diff
@@ -29,18 +29,21 @@ const userFn = getFunction(USER_FUNCTION_URI);
 module.exports = async ({headers, payload}) => {
   if (DEBUG) {
     console.log('==System Function Start==');
-    console.log(`HEADERS: ${JSON.stringify(headers)}`);
+    console.log(`HEADERS: ${headers}`);
```

Comment:
Question: Are these headers the riff message headers or are they a plain ol JS object?